soccer/Robot.cpp

```diff
@@ -162,8 +161,10 @@ void OurRobot::resetForNextIteration() {
 }
 
 void OurRobot::resetMotionConstraints() {
+    _rotationConstraints = Planning::RotationConstraints();
     _motionConstraints = MotionConstraints();
-    _motionCommand = Planning::MotionCommand();
+    _motionCommand = make_unique<Planning::WorldVelTargetCommand>(Geometry2d::Point(0,0));
```

Comment:
I thought about that but would it really be different from a worldvel command of nothing. Theres enough madness with the types of commands already. I used nullptr before but then thats more cases that we need to do nullptr checks.